app/views/admin/articles/_individual_article.html.erb

```diff
@@ -4,19 +4,29 @@
   }
 </style>
 
-<div class="card my-3" data-controller="article" data-article-id="<%= article.id %>">
+<div class="card my-3"
+     data-controller="article"
+     data-article-bg-highlighted-class="bg-highlighted"
+     data-article-border-highlighted-class="border-highlighted">
```

Comment:
we tell Stimulus what are the CSS classes for the various CSS class properties, this way if `bg-highlighted` changes in the CSS, there's only one place to change it as well